networkx/algorithms/euler.py

```diff
@@ -224,8 +242,8 @@ def has_eulerian_path(G, source=None):
         - at most one vertex has out_degree - in_degree = 1,
         - at most one vertex has in_degree - out_degree = 1,
         - every other vertex has equal in_degree and out_degree,
-        - and all of its vertices with nonzero degree belong to a
-          single connected component of the underlying undirected graph.
+        - and all of its vertices belong to a single connected
+        component of the underlying undirected graph.
```

Comment:
```suggestion
        - and all of its vertices belong to a single connected
          component of the underlying undirected graph.
```

I think this is necessary to get the list to render correctly in the generated html!